Demo/Demo/FeatureViewControllers/PayPalDemoViewController.swift

```diff
@@ -51,7 +65,7 @@ class PayPalDemoViewController: FeatureBaseViewController {
 
     // MARK: - PayPal Module Integration
 
-    @objc func didTapPayPalButton() {
+    @objc func paymentButtonTapped() {
         checkoutWithPayPal(orderID: orderID ?? "")
```

Comment:
Instead of coalescing orderID to nil, should we do a guard here and update title to be `Failed: missing orderID.` if it's nil to make the behavior more like card demo? :smile:

Reply:
That's a _great_ callout. Updated! 😄